docs/cameras/camera.rst

```diff
@@ -11,3 +11,7 @@ Snapshot Camera
 ---------------
 .. autoclass:: zoloto.cameras.camera.SnapshotCamera
     :members:
+
+Base Camera
```

Comment:
Might be nicer to move this to `docs/cameras/index.rst` rather than here?

Reply:
1975f5a